ibm_catalog.json

```diff
@@ -304,6 +304,13 @@
 							"default_value": "__NULL__",
 							"description": "The CRN of an existing KMS instance to use in this solution. If not set, a new KP instance is provisioned.",
 							"required": false
+						},
+						{
+							"key": "existing_en_instance_crn",
```

Comment:
I think there are a couple of other new inputs in [stack_definition.json](https://github.com/terraform-ibm-modules/dev-rag/pull/7/commits/a670acf7d37a2625fb71fb26791c1a0ef20c483b#diff-47513878fa1687ae846161d0fd5e96a62442127ddb151bd015cc9881b065144f) that you missed